setup.py

```diff
@@ -16,7 +16,7 @@
 install_requires = [
      'enaml>=0.10',  # must install pyqt or pyside
      'twisted',
-     'enamlx',  # use pip install git+https://github.com/frmdstry/enamlx.git
+     'enamlx',  # use pip install git+https://github.com/frmdstryr/enamlx.git
```

Comment:
the git repo is no longer needed, enamlx is now on pypi so it can be removed completely